pwndbg/commands/probeleak.py

```diff
@@ -19,16 +19,13 @@
 
 
 def find_module(addr):
-    mod_filter = lambda page: page.vaddr <= addr < page.vaddr + page.memsz
+    mod_filter = lambda page: page.vaddr <= addr <= page.vaddr + page.memsz
     pages = list(filter(mod_filter, pwndbg.vmmap.get()))
 
     if not pages:
         return None
 
-    if len(pages) > 1:
-        print(message.warn('Warning: There is more than one page containing address 0x%x (wtf?)', addr))
-
-    return pages[0]
+    return pages[-1]
```

Comment:
With "<=" we can find ptrs to the end of the page, which may be missed in previous version when some page doesn't have adjacent neighbor

Reply:
Not sure I get it, can you give an example?
Afaik if you have a page from address A to B that means it occupies addresses [A, B) - not including B. So I guess the `<=` change above is wrong?

Reply:
Yes, page is in [A,B), but is we leak pointer to the end of the page, so to the B, we can still show it in the probeleak.
And pages[-1] because when two pages are aligned it is better to show the second one.